### DISCUSSION TITLE
How to render w2grid multiple times based on class

### DISCUSSION THREAD

**Discussion by foxpcteam** [vitmalina/w2ui#2313] [Q&A]
Hi,
I did not find any example of how to render the same widget multiple times based on class.
* the data of each widget will be updated programmatically.

If i have HTML with 2 divs that have same class like:
```
<div class="main" style="width: 100%; height: 800px;">/div>
<div class="main" style="width: 100%; height: 800px;">/div>
```
I want that each div will be converted to w2grid.

But now, when we use
`grid.render('.main');`

It will take only the first div and render it to the w2grid.

So the question is if w2ui can handle multiple rendering based on class?

**Comment by vitmalina** [comment]
A single grid can be rendered to a single div. You need to create 2 grids if you want to render them into 2 divs
